### ISSUE TITLE
Pls verify New creator too instead of verify big project only!!

### ISSUE THREAD

**Issue by RichAnimals** [Cardano-NFTs/policyIDs#1830]

**Comment by CardanoVerseProject** [comment]
I second this statement. It's not even about "new creators".

People who are real artists and passionate about drawing, and they only plan to mint a few NFTs in their lifetime, should have equal priority as everyone else. The big projects are tended to more frequently only because they do a lot of marketing and PR.

I know some amateur artists just like me and we'd like to mint a few but very detailed artwork NFTs instead of hundreds of them that take 5 minutes to draw. No offense to those projects; I respect their vision and I know they have all have purposes.

It's just that the community should be equal to everyone, quoting Charles Hoskinson. Therefore, these "PR" and "Twitter proof" criteria should be relaxed a bit because not a lot of artists want to post their artwork that they spent days drawing without launching actual projects. Again, no offense to those NFTs.

Cheers.

**Comment by RichAnimals** [comment]
i agreed with u..some of the artist might be released quality artwork every week..In the long run, they doing great job what short-term cannot see it.

**Comment by SickLead** [comment]
Totally agree, they should respect the waiting line! In programmers terms "FIFO" !

**Comment by todiadiyatmo** [comment]
I agree, why the verification took so long ? other marketplace do it rather quickly

**Comment by cardanomushrooms** [comment]
This is all true. Been weeks and all I see is that they only approving big projects. The thing is these big projects are not even in que and just get approved right away while smaller projects like us are still waiting for weeks to be approved. This is just not fair.

**Comment by firdauszahrin** [comment]
Please be fair and square.

**Comment by VectorGameStudio** [comment]
yeah, I only see projects created like few hours ago gets accepted, only a matter of mona Imo.
